src/app/modules/email-modal/email-modal.component.ts

```diff
@@ -217,14 +219,15 @@ export class EmailModalComponent implements OnInit, OnDestroy, AfterContentInit
                             duration: 2000,
                         });
                         this.commonService.inboxRefreshEvent();
+                        broadcast_send({ 'any':'value' });
```

Comment:
in that case rename the function to
notifyInbox()  no need of sending any value